pumpkin/src/world/block.rs

```diff
@@ -0,0 +1,9 @@
+use serde::Deserialize;
+
+#[derive(Debug, Deserialize)]
+pub struct Block {
+    pub x: i32,
+    pub y: i32,
+    pub z: i32,
+    pub id: i32,
```

Comment:
For now an id is fine, I think in the future we should move to an enum

Reply:
Read the comment in chunk.rs!